os-autoinst-obs-auto-submit

```diff
@@ -18,10 +18,33 @@ XMLSTARLET=$(command -v xmlstarlet || true)
 # shellcheck source=/dev/null
 . "$(dirname "${BASH_SOURCE[0]}")"/_common
 
+check_project_exists() {
+    # ex: openSUSE:Backports:SLE-15-SP6
+    local prj="$1"
+    local rc=0
+    if [[ -z "$prj" ]]; then
+        echo "Error: No project provided." >&2
+        return 1
+    fi
+    # dont need the XML output for checking existence.
+    # if $prj does not exist return 404
```

Comment:
under which circumstances would `osc dists` return a project that does not exist?

Reply:
`osc dists` is unlikely to not retun a project unless the `submit_target_extra_project` is arbitrary.
`osc meta` will return 404 if the `:Update` project is not found